nncf/api/composite_compression.py

```diff
@@ -258,10 +267,6 @@ def build_controller(self, model: ModelType) -> CompositeCompressionAlgorithmCon
          algorithm-specific compression during fine-tuning.
         :return: The instance of the `CompositeCompressionAlgorithmController`.
         """
-        composite_ctrl = CompositeCompressionAlgorithmController(model)
-        for builder in self.child_builders:
-            composite_ctrl.add(builder.build_controller(model))
-        return composite_ctrl
```

Comment:
Could you explain why you deleted the implementation of the `build_controller` method.

Reply:
`CompositeCompressionAlgorithmController`  and `CompositeCompressionAlgorithmBuilder` are interfaces. This classes can't be instantiated because not all abstract methods are implemented. The code that was deletedd was never executed. Method `build_controller` implemented in  `TFCompositeCompressionAlgorithmBuilder` and `PTCompositeCompressionAlgorithmBuilder`.

Reply:
Thank you for the answer. I understand, Should we remove remove ` def build_controller(self, model: ModelType) -> CompositeCompressionAlgorithmController:` from the CompositeCompressionAlgorithmBuilder abstraction class. It looks like the `get_transformation_layout` method has the same issue.

Reply:
Fixed the same issue for `get_transformation_layout`.

Reply:
Thanks. I just want to understand. Should we override all methods of the abstraction class in the inhered class if the inhered class is abstract too? It looks like you should remove `build_controller` and `transformation_layout` methods or add `apply_to` method to the CompositeCompressionAlgorithmController for consistency.